src/pages/Favorites/Favorites.test.js

```diff
@@ -8,4 +8,29 @@ describe('<FavoritesPage>', () => {
     const { container } = render(<FavoritesPage>{'Test'}</FavoritesPage>);
     expect(container.querySelector('div')).toBeValid();
   });
+
```

Comment:
This could be improved by trying to come up with more meaningful tests. Beyond stating that "a container will appear", we can improve by testing for "An element with the word 'Favorites' will appear", or "x-number of videocards will appear"